Cargo.toml

```diff
@@ -63,6 +63,9 @@ test-engines-panic = [
   "engine_test/test-engine-kv-panic",
   "engine_test/test-engine-raft-panic",
 ]
+bcc-iosnoop = [
```

Comment:
It does have a platform-specific-dependencies, see https://github.com/tikv/tikv/pull/9153/files/42aac47c4a3d0edf7720d4c6057233f892cc8eb0#diff-5a645a821a65175c939f78c4cf3491a3ef5719dedd9b802f7c0fec00cc95a711R20

But for Linux version which doesn't support bcc or bcc is not installed, the tikv build will fail. So I provide a feature to enable/disable so that older Linux can build tikv normally.

Reply:
make sense. Do we need to enable the feature by default in the Makefile?

Reply:
let me ask the linux version of our CI. If that, `dist_release` can be built with bcc-iosnoop by default. But for `release`, I still suggest to disable by default, otherwise, contributors may suffer from build problems.

Reply:
Sadly, the CI is still bounded on CentOS7 of 3.10 kernel. But we can apply for upgrading the kernel version of a small group of machines.